app/src/main/java/nu/yona/app/api/manager/NotificationManager.java

```diff
@@ -38,6 +38,16 @@
 
     void getMessage(final int itemsPerPage, final int pageNo, boolean isUnreadStatus, final DataLoadListener listener);
 
+
+    /**
+     * Gets message.
+     *
+         * @param urlForMessagesFetch the items per page
+     * @param isUnreadStatus       the page no
+     * @param listener     the listener
+     * @param isProcessUpdate
```

Comment:
What does ``isProcessUpdate`` mean?

Reply:
This is one of the questions I had for mobiquity. I even don't see that code getting called. I didnt disturb the code as every thing is working fine.
if this is true, they are calling a post services with getYonaPreocess url with empty message body and properties.
In other places they are using same post call to set notification message as markRead.

Reply:
I see. Messages in Yona can have actions, e.g. "mark read" or "accept" (on a buddy connect request). To trigger such an action, a ``POST`` needs to be done.

In the past, the app needed to do the do the action "process" on certain messages. Later, we have moved this functionality to the server. They have disabled the mechanism but not cleaned up the code. Please clean it up, as we will never revert to the old way of working.

Reply:
I will update this in my next commit.

Reply:
OK